docs/source/v1/BGC/simulation_data/index.rst

```diff
@@ -0,0 +1,64 @@
+***************
+Simulation Data 
+***************
+
+The E3SMv1 simulation data is available on **NERSC HPSS**.
+
+All native model output data has also been archived on **NERSC HPSS** using `zstash <https://e3sm-project.github.io/zstash>`_.
+
+**If you have an account on NERSC**, you can retrieve the data locally or remotely using Globus.
+
+To download simulation data locally on a NERSC machine: ::
+
+   zstash extract --hpss=<HPSS path in table>
+
+To download simulation data remotely using the zstash Globus interface: ::
+
+   zstash extract --hpss=globus://nersc/<HPSS path in table>
+
+or ::
+
+   zstash extract --hpss=globus://9cd89cfd-6d04-11e5-ba46-22000b92c6ec/<HPSS path below>
+
+Note that the data management tool `zstash <https://github.com/E3SM-Project/zstash>`_ is available from the `E3SM-Unified <https://github.com/E3SM-Project/e3sm-unified>`_ conda environment. An example of retrieving all **cam.h0** (monthly atmosphere output files) between **years 0030 and 0049** for the piControl simulation at NERSC locally is demonstrated as below in two steps:
+
+1. To activate E3SM-Unified environment by:
+   ::
+
+    source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_pm-cpu.sh
+
+2. To retrieve files with zstash command:
+   ::
+
+    zstash extract --hpss=/home/projects/e3sm/www/BGC/E3SMv1/E3SM_1_1_piControl "*.cam.h0.00[3-4]?-??.nc"
+    # (TODO: run an actual zstash command to make sure it works as an example)
```

Comment:
I will find an example `zstash` command to put here that works on this particular data set.